Calibration/PPSAlCaRecoProducer/python/PPSAlCaRecoProducer_cff.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+hltFilter_express = copy.deepcopy(hltHighLevel)
+hltFilter_express.TriggerResultsTag = cms.InputTag("TriggerResults","","HLTX")
+hltFilter_express.HLTPaths = ['HLT_PPSMaxTracksPerArm1_v1']
```

Comment:
If I understand it correctly, this is the exact triggerbit? Maybe here you could either use a wildcard or even better, include a key in the AlCaRecoTriggerBits tag, as done in here: https://github.com/cms-sw/cmssw/blob/master/Calibration/TkAlCaRecoProducers/python/ALCARECOSiStripCalCosmics_cff.py#L7

Reply:
I second the idea to put it in the AlCaRecoTriggerBits tag!

Reply:
@grzanka

Reply:
@grzanka since this is done now, please resolve this comment